swagger-spec/draft_registrations/draft_registrations_list.yaml

```diff
@@ -0,0 +1,242 @@
+                      related:
+                        href: https://api.osf.io/v2/schemas/registrations/5e795fc0d2833800195735d0/
+                        meta: { }
+                    data:
+                      id: 5e795fc0d2833800195735d0
+                      type: registration-schemas
+                  provider:
+                    links:
+                      related:
+                        href: https://api.osf.io/v2/providers/registrations/osf/
+                        meta: { }
+                    data:
+                      id: osf
+                      type: registration-providers
+                  affiliated_institutions:
+                    links:
+                      related:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/institutions/
+                        meta: { }
+                      self:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/relationships/institutions/
+                        meta: { }
+                  contributors:
+                    links:
+                      related:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/contributors/
+                        meta: { }
+                  bibliographic_contributors:
+                    links:
+                      related:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/bibliographic_contributors/
+                        meta: { }
+                  license:
+                    data:
+                  subjects:
+                    links:
+                      related:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/subjects/
+                        meta: { }
+                      self:
+                        href: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/relationships/subjects/
+                        meta: { }
+                links:
+                  self: https://api.osf.io/v2/draft_registrations/626170854968470203611e2c/
+          meta:
+            total: 1
+            per_page: 10
+            version: '2.20'
+          links:
+            self: https://api.osf.io/v2/draft_registrations/
+            first:
+            last:
+            prev:
+            next:
+
+post:
+ summary: Create a Draft Registration
+ description: >-
+   This creates a Draft Registration that can be used to edit and register research.
+
+   Draft Registrations contain Registration questions that will become part of the Registration.
+   A Registration is a frozen version of the project that can never be edited or deleted, but can be withdrawn.
+
+   If you based your Draft Registration on a Project, your original Project remains editable but will now have
+   the Draft Registration linked to it. However if you are using this endpoint no Project will be linked.
```

Comment:
Can't the user use the `branched_from` relationship to define a link to a project using this endpoint?

Reply:
Yep I added that info!